### PR TITLE
feat: PDP basic info

### PR DESCRIPTION
**Pull request by mikemurray**
Issue: #22 ( depends on #21 )
Impact: **minor**
Type: **feat**

## Issue

Create basic info on the PDP page with price, vendor, and description fields.

## Solution 

Create a component that can

- Display price, vendor, description if the data is passed to the component.

## Test

- Start the app and go to `http://localhost:4000/product/1234`
- See the following

reaction_shop

### CODE REVIEW

File: src/components/ProductDetailInfo/ProductDetailInfo.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import Grid from "material-ui/Grid";
+
+/**
+ * Product detail basic info fields
+ * @class ProductDetailTitle
+ */
+class ProductDetailInfo extends Component {
+  static propTypes = {
+    /**
+     * Prodcut description
```
Comment (willopez) [inline src/components/ProductDetailInfo/ProductDetailInfo.js]: Typo here, should be Product

File: src/components/ProductDetailInfo/ProductDetailInfo.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Product vendor
+     */
+    vendor: PropTypes.string
+  }
+  render() {
+    const { priceRange, vendor, description } = this.props;
+
+    // Render all props are undefined then skip rendering component
```
Comment (willopez) [inline src/components/ProductDetailInfo/ProductDetailInfo.js]: I believe this is a typo, it should be something like: "If all props are undefined ..."

File: src/pages/product.js [hunk 3]
```diff
@@ -7,6 +7,9 @@ import withShop from "containers/shop/withShop";
 import Layout from "components/Layout";
 import ProductDetail from "components/ProductDetail";
 
+// TODO: Data will eventually come from GraphQL
+import sampleData from "components/ProductDetail/sampleData";
```
Comment (willopez) [inline src/pages/product.js]: What are your thoughts on using a jest convention for naming mocked data. I like using a folder named `__mocks` that would contained a file name `product.js` with the product object? and we can use those mocks in jest tests.

**Comment by mikemurray** [comment]
@will made those changes, also moved the sample data to `__mocks__` since it'll only be needed for tests once we hook the PDP page to real data.

**Comment by spencern** [comment]
@mikemurray looks like you tagged the wrong @willopez in this issue :)

**Comment by mikemurray** [comment]
Yup seems that way

**Pull request merged**